lib/assets.js

```diff
@@ -69,9 +69,10 @@ TradeOfferManager.prototype._addDescriptions = function(items, callback) {
 	this._requestDescriptions(descriptionRequired, (err) => {
 		if (err) {
 			callback(err);
-		} else {
-			callback(null, this._mapItemsToDescriptions(null, null, items));
+			return;
```

Comment:
You should probably either change this to `return callback(err);` or split line 152 into two separate lines like this.